app/assets/v2/js/quadraticlands/mission/diplomacy/room.js

```diff
@@ -0,0 +1,294 @@
+document.addEventListener('DOMContentLoaded', function() {
+
+  console.debug('DIPLOMACY ROOM');
+
+  // show and hide arrownav when #trollbox is / is not in visible area
+  const arrownav = document.getElementById('arrownav');
+  var observer = new IntersectionObserver(function(entries) {
+    if (entries[0].isIntersecting === true)
+      arrownav.classList.remove('hide');
+    else
+      arrownav.classList.add('hide');
+  }, { threshold: [0] });
+
+  observer.observe(document.querySelector('#trollbox'));
+
+
+  //
+  //
+  document.refresh_page = function(url) {
+    var keys = [/* '.diplomacy-room-members', '.diplomacyvouchebar', */'.entries'];
```

Comment:
@thelostone-mc i cant get these UI elements to refresh on the UI .  `.entires` works but the other ones dont